src/address.rs

```diff
@@ -1,8 +1,23 @@
-use crate::keys::Diversifier;
+use crate::keys::{DiversifiedTransmissionKey, Diversifier};
 
 /// A shielded payment address.
+///
+/// # Examples
+///
+/// ```
+/// use orchard::keys::{SpendingKey, FullViewingKey};
+///
+/// let sk = SpendingKey::from_bytes([7; 32]).unwrap();
+/// let address = FullViewingKey::from(&sk).default_address();
+/// ```
 #[derive(Debug)]
 pub struct Address {
     d: Diversifier,
-    pk_d: (),
+    pk_d: DiversifiedTransmissionKey,
+}
+
```

Comment:
```rust
impl From<(Diversifier, DiversifiedTransmissionKey)> for Address {
    pub fn from((d, pk_d): (Diversifier, DiversifiedTransmissionKey)) -> Self {
        Self::from_parts(d, pk_d)
    }
}

```

Reply:
We discussed this in Discord and agreed this particular case wouldn't be included.